### ISSUE TITLE
S3444 Add missing name in the message for indexers

### ISSUE THREAD

**Issue by KarbyCZ** [SonarSource/sonar-dotnet#3225]
### Description

I am getting a warning S3444 on my compound interface `IDynamicList<T>` that implements `IList<T>` and `IReadOnlyList<T>`. The message is not actionable, the member name is missing:

> Rename or add member '' to this interface to resolve ambiguities.

### Repro steps

```csharp
public interface IDynamicList<T> : IList<T>, IReadOnlyList<T>
{
}
```

### Expected behavior

The message should always provide a name of the conflicting interface member. I believe it is the indexer in this case.

### Actual behavior

The conflicting interface member name is blank in the message. I believe it is the indexer. Adding a new indexer stops the warning:

```csharp
public interface IDynamicList<T> : IList<T>, IReadOnlyList<T>
{
    new T this[int index] { get; set; }
}
```

### Related information

* SonarAnalyzer.CSharp version 8.5.0.15942
* VS2019 version 16.4.6
* MSBuild version 16.4.0, dotnet SDK version 3.1.102
* Windows 10, build 1909

[Rule S3444 description](https://jira.sonarsource.com/browse/SLVS-673)

**Comment by pavel-mikula-sonarsource** [comment]
Hi KarbyCZ,
Thank you for reporting this. It was a known issue, because indexers doesn't have name, but we can improve the message to fix this.
